### PR TITLE
Update install.md

### PR DESCRIPTION
**Pull request by timmy61109**
#### What type of PR is this?

/kind other

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: install.md [hunk 1]
```diff
@@ -181,18 +181,23 @@ apt update
 apt install -y -t buster-backports libseccomp2 || apt update -y -t buster-backports libseccomp2
```
Comment (haircommander) [inline install.md:181]: can this section be updated with sudo if the section below is?

**Comment by TomSweeneyRedHat** [comment]
LGTM
once @haircommander 's concerns are addressed.

**Comment by sohankunkerkar** [comment]
@timmy61109 do you have the bandwidth to address @haircommander's comments and make this PR ready to merge?

**Comment by timmy61109** [comment]
> @timmy61109 do you have the bandwidth to address @haircommander's comments and make this PR ready to merge?

I'm sorry, I don't really know what you mean by "bandwidth"?Can you explain this to me?

In addition, to clarify the above modification, I just refer to the official website's installation method, but the result is unsuccessful. I checked the official website and GitHub WiKi wrote differently, and then I read other people's articles on the internet, and I found that the method is actually the same as that written by GitHub WiKi, and after I organized it, I succeeded.

What I don't understand is why the `sudo` command is not provided, after all, not everyone can install as root, and the Ubuntu server version is also for general users.

Translated with www.DeepL.com/Translator (free version)

**Comment by haircommander** [comment]
>   I'm sorry, I don't really know what you mean by "bandwidth"?Can you explain this to me?

bandwidth in this case is "capacity" as in: do you have enough time and energy to address what I've said

> What I don't understand is why the sudo command is not provided, after all, not everyone can install as root, and the Ubuntu server version is also for general users.

We usually assume CRI-O will be run and installed as root and by a user who has root permissions, which is why we didn't really add sudo before. I am fine adding the sudo, but we should do it consistently through the whole document

**Comment by timmy61109** [comment]
> > I'm sorry, I don't really know what you mean by "bandwidth"?Can you explain this to me?
> 
> bandwidth in this case is "capacity" as in: do you have enough time and energy to address what I've said
> 
> > What I don't understand is why the sudo command is not provided, after all, not everyone can install as root, and the Ubuntu server version is also for general users.
> 
> We usually assume CRI-O will be run and installed as root and by a user who has root permissions, which is why we didn't really add sudo before. I am fine adding the sudo, but we should do it consistently through the whole document

我了解了，我後續會找時間來修改，請問目前有多少篇呢？

I understand, I will find time to revise it later, may I ask how many articles there are at present?

**Comment by haircommander** [comment]
> I understand, I will find time to revise it later, may I ask how many articles there are at present?

you can just focus on `install.md` for now. thanks!

**Comment by timmy61109** [comment]
> > I understand, I will find time to revise it later, may I ask how many articles there are at present?
> 
> you can just focus on `install.md` for now. thanks!

OK

**Pull request closed without merging**